src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+:root {
+  --maincolor: #00acdc;
+  --header-height: 60px;
+  --header-margin: 50px;
+}
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  .logo {
+    display: flex;
+    align-items: center;
+    margin-left: var(--header-margin);
+  }
```

Comment:
CSS rules for `.logo` should not be nested inside `.header`. In standard CSS, nesting is not allowed. You may be using a preprocessor like SASS, but the resulting CSS should not have nested rules.